src/goto-symex/Makefile

```diff
@@ -35,6 +35,7 @@ SRC = auto_objects.cpp \
       symex_target.cpp \
       symex_target_equation.cpp \
       symex_throw.cpp \
+      symex_complexity.cpp \
```

Comment:
commit message for "Initial commit of symex complexity work": "This means that our trace will miss context" -> "This means we won't be able to generate traces that would use the discarded path"

Reply:
Also we don't generate partial traces because of this, we generate normal traces but may fail to spot some which otherwise would have been generated